drivers/at/at.c

```diff
@@ -31,10 +31,9 @@ int at_dev_init(at_dev_t *dev, uart_t uart, uint32_t baudrate, char *buf, size_t
 {
     dev->uart = uart;
     isrpipe_init(&dev->isrpipe, buf, bufsize);
-    uart_init(uart, baudrate, _isrpipe_write_one_wrapper,
-              &dev->isrpipe);
 
-    return 0;
+    return uart_init(uart, baudrate, _isrpipe_write_one_wrapper,
+	    &dev->isrpipe);
```

Comment:
final nit: correct indention, i.e., the `&dev` should align with the first param (`uart`) in the line above.

Reply:
done